### PR TITLE
Update copy-metadata-files response

### PR DESCRIPTION
**Pull request by Dhwaniartefact**
Connected to https://github.com/archivematica/Issues/issues/1656

### CODE REVIEW

File: dev-manual/api/api-reference-archivematica.rst [hunk 1]
```diff
@@ -484,9 +484,9 @@ Response definitions:
 
 ==================   ==========================================================
 
-``message``          "Approval successful."
+``error``            "false"
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]:
```suggestion
``error``            Boolean true or false.
```

File: dev-manual/api/api-reference-archivematica.rst [hunk 2]
```diff
@@ -484,9 +484,9 @@ Response definitions:
...
 
-``reingest_uuid``    UUID of the reingested transfer.
+``message``          "Metadata files added successfully."
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]:
```suggestion
``message``          Success message or reason for the error.
```

**Review by replaceafill — CHANGES_REQUESTED** [review]
I think we should document it in a more generic way. Alternatively, see https://www.archivematica.org/en/docs/archivematica-1.15/dev-manual/api/api-reference-archivematica/#validate which also includes examples of the different responses you might get on failures.

File: dev-manual/api/api-reference-archivematica.rst (hunk 2 above)
Comment (Dhwaniartefact) [reply dev-manual/api/api-reference-archivematica.rst]: Thanks for the suggestion. Let me work on it.

File: dev-manual/api/api-reference-archivematica.rst [hunk 3]
```diff
@@ -484,13 +484,45 @@ Response definitions:
...
 ==================   ==========================================================
 
 
+Usage example::
+
+  curl -X POST \
+  http://my_archivematica_instance.archivematica.org/api/ingest/copy_metadata_files/ \
+  -H 'Authorization: ApiKey [_your_username_]:[_your_ApiKey_]' \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  -d 'sip_uuid=a7f1f36c-55c1-4fb7-a898-76d102e847dc&source_paths%5B%5D='''NmRjYTgxNDAtMDg2NC00MzE5LTg2ZDctNTg0ZjZiZTY4N2EzOi9ob21lL2FyY2hpdmVtYXRpY2EvYXJjaGl2ZW1hdGljYS1zYW1wbGVkYXRhL2lzc3VlX3RlbXBsYXRlLm1k''''
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]: Indentation on the page is not very consistent, but I noticed the `Response examples` below use 4 spaces indentation. Also extra indentation on the `curl` flags might be useful.

```suggestion
    curl -X POST \
        http://my_archivematica_instance.archivematica.org/api/ingest/copy_metadata_files/ \
        -H 'Authorization: ApiKey [_your_username_]:[_your_ApiKey_]' \
        -H 'Content-Type: application/x-www-form-urlencoded' \
        -d 'sip_uuid=a7f1f36c-55c1-4fb7-a898-76d102e847dc&source_paths%5B%5D='''NmRjYTgxNDAtMDg2NC00MzE5LTg2ZDctNTg0ZjZiZTY4N2EzOi9ob21lL2FyY2hpdmVtYXRpY2EvYXJjaGl2ZW1hdGljYS1zYW1wbGVkYXRhL2lzc3VlX3RlbXBsYXRlLm1k''''
```

File: dev-manual/api/api-reference-archivematica.rst [hunk 4]
```diff
@@ -484,13 +484,45 @@ Response definitions:
...
+Response examples
+
+200 OK::
+
+    {
+      "error": false,
+      "message": "Metadata files added successfully."
+    }
+
+400 Bad Request ::
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]:
```suggestion
400 Bad Request::
```

File: dev-manual/api/api-reference-archivematica.rst [hunk 5]
```diff
@@ -484,13 +484,45 @@ Response definitions:
...
+  curl -X POST \
+  http://my_archivematica_instance.archivematica.org/api/ingest/copy_metadata_files/ \
+  -H 'Authorization: ApiKey [_your_username_]:[_your_ApiKey_]' \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  -d 'sip_uuid=a7f1f36c-55c1-4fb7-a898-76d102e847dc&source_paths%5B%5D='''NmRjYTgxNDAtMDg2NC00MzE5LTg2ZDctNTg0ZjZiZTY4N2EzOi9ob21lL2FyY2hpdmVtYXRpY2EvYXJjaGl2ZW1hdGljYS1zYW1wbGVkYXRhL2lzc3VlX3RlbXBsYXRlLm1k''''
+
+
+Response examples
+
+200 OK::
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]: This should be `201`. Please update the `OK` part too.

File: dev-manual/api/api-reference-archivematica.rst [hunk 6]
```diff
@@ -484,13 +484,45 @@ Response definitions:
...
+    {
+      "error": true,
+      "message": "sip_uuid and source_paths[] both required."
+    }
+
+500 Internal Server Error::
+
+    {
+      "error": true,
+      "message": "sip_uuid and source_paths[] both required."
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]: Reading [the code](https://github.com/artefactual/archivematica/blob/d42c61206b0e82af9f6a39386b801c2d39a0e827/src/dashboard/src/components/filesystem_ajax/views.py#L963-L970) I don't think this error message is attached to the `500` error.

```suggestion
      "message": "Location 6dca8140-0864-4319-86d7-584f6be687a3 is not associated with this pipeline"
```

**Review by replaceafill — CHANGES_REQUESTED** [review]

File: dev-manual/api/api-reference-archivematica.rst (hunk 6 above)
Comment (Dhwaniartefact) [reply dev-manual/api/api-reference-archivematica.rst]: Let me go through more exceptions.

File: dev-manual/api/api-reference-archivematica.rst (hunk 5 above)
Comment (Dhwaniartefact) [reply dev-manual/api/api-reference-archivematica.rst]: oops

**Review by replaceafill — APPROVED** [review]

**Pull request merged**